validator/core/config.py

```diff
@@ -35,6 +35,7 @@ class Config:
     set_metagraph_weights_with_high_updated_to_not_dereg: bool
     testnet: bool = os.getenv("SUBTENSOR_NETWORK", "").lower() == "test"
     debug: bool = os.getenv("ENV", "prod").lower() != "prod"
+    default_organic_task_history_hours: int = 5
```

Comment:
What does this mean?

Reply:
The get organic tasks endpoint takes an argument of how many hours to look back for. This is the default value. 
Open to suggestions for a better name

Reply:
Why is this is config? This belongs on the endpoint, not in config

Reply:
good point. Moved to the endpoint